src/server/routes.go

```diff
@@ -93,10 +94,24 @@ func (s *Server) handleManifest(c *router.Context) {
 				"sizes": "64x64",
 				"type":  "image/png",
 			},
+			{
+				"src":   s.BasePath + "/static/graphicarts/favicon-144.png",
+				"sizes": "144x144",
+				"type":  "image/png",
+			},
+			{
+				"src":   s.BasePath + "/static/graphicarts/favicon.svg",
+				"sizes": "any",
+				"type":  "image/svg",
+			},
 		},
 	})
 }
 
+func (s *Server) handleServiceWorker(c *router.Context) {
+	http.ServeFile(c.Out, c.Req, "src/assets/javascripts/sw.js")
```

Comment:
is it guaranteed to work for a compiled binary (which embeds assets)?

Reply:
Yeah I see. Do you have any suggestion on how should I handle this service worker file?